### ISSUE TITLE
Calc with internal parentheses being deleted.

### ISSUE THREAD

**Issue by Hazzard13** [jaraco/cssutils#5]
Pardon the unusually high specificity of this bug, but this has been a serious blocker for me, and has cost me most of a day.

Basically, from my research I've found that any line of CSS including parentheses _inside_ a calc statement will be removed. To test, all I'm doing with my CSS currently is parsing and printing it (Using the current v2.2.0 version, ofc):

```
stylesheet = cssutils.parseString(css_contents)
print(stylesheet.cssText.decode("utf-8"))
```

In my findings, even lengthy calcs like this will be left alone: `font-size: calc(15px * 100vw - 600px / 1320);`, but the moment any brackets are introduced it will fail, like this line: `font-size: calc((100vw - 600px) * 10);`.

Functions like this are supposedly meant to be handled according to the [documents linked in the README](https://drafts.csswg.org/css-values-4/#funcdef-calc), look at Example 29 for particulars. Regarding that same example, I've also tried nested calcs in the same fashion, to no avail: `font-size: calc(calc(100vw - 600px) * 10);`.

Basically what I'm asking, is if there's any way for this bug to be fixed or worked around? Ideally, I'd love something I can do today, to either skip validating these lines or to find a way to get them to pass without changing the meaning.

**Comment by Hazzard13** [comment]
I've taken a good crack at this myself, but been unable to make too much headway, due to simply not being familiar with the codebase. Regardless, here's the fruit of my labour and what I assume needs to be done here.

https://github.com/jaraco/cssutils/blob/cdffe59ac62e6e7dd0240e54f4952f271df46230/cssutils/css/cssvalue.py#L1208-L1238

https://github.com/jaraco/cssutils/blob/cdffe59ac62e6e7dd0240e54f4952f271df46230/cssutils/css/value.py#L788-L850

Basically, somehow CalcValue and CSSCalc needs to recursively refer to themselves, with an Operator of either "(" or "calc(". The actual handling of calc as it currently exists is perfectly adequate at handling all other cases, at least in my testing and experience. Additionally, this particular check for a closing bracket probably needs to be removed or reworked: 

https://github.com/jaraco/cssutils/blob/cdffe59ac62e6e7dd0240e54f4952f271df46230/cssutils/css/cssvalue.py#L1230

I've been unable to get anything to build that functions substantially different, but hopefully someone on the team can pick this up and move it further. If that's you, thanks so much for your time and effort in this library.

**Comment by jaraco** [comment]
I've adopted maintenance of this package, but unfortunately, I don't know the codebase much at all. I appreciate the work you've done and welcome you or others to do more.
